app/src/main/res/values-w1240dp/dimens.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+      <dimen name="fab_margin">200dp</dimen>
```

Comment:
Same here - focus on default values, if you want to support other devices you can handle it later, but then check carefully which values are correct for specific screen size.